unit_tests/data/configs/testDB.conf

```diff
@@ -2,6 +2,9 @@
 mission_name = testDB
 rootdir = /home/myles/dbprocessing/test_DB
 incoming_dir = L0
+errordir = error
+codedir = code
+inspectordir = inspector
```

Comment:
These shouldn't be necessary. They only had to be added because of removing the optional keys in addFromConfig; I restored that. But forgot to restore this. I'll remove these in the next rebase.

Reply:
We do use errordir/codeir and inspectordir,  intensively.   The testDB was not able to read our config files, so I thought I needed to add them here.   If you can make them optional that will be fine. 

BTW, AddFromConfig.py still gives me a hard time when I add a new instrument to the satellite.   There is something wrong in the mission section I think.

Reply:
The directories would still exist and all the functions that return them will work (actually, the columns are still created even), it'll just be the defaults, and only if you don't specify (if you specify, that overrides defaults.)

Was addFromConfig giving you trouble with a new instrument before or is this new since I made these changes? If it was happening before, can you open an issue and I can look into it?

Reply:
`AddFromConfig` needed to be revamped from the python version 2 we had.
  
Once you merged,  I'll test it and if there is something wrong, I will open a new issue.  Thanks!